src/styles/blocks/card.scss

```diff
@@ -0,0 +1,122 @@
+.card-menu {
+  --count-column: 1;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--count-column) 200px);
+  grid-template-rows: repeat(2, 406px);
+  gap: 48px 46px;
+
+  @media (min-width: 488px) {
+    --count-column: 2;
+  }
+
+  @media (min-width: 768px) {
+    --count-column: 3;
+  }
+
+  @media (min-width: 1024px) {
+    --count-column: 4;
+  }
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  background-color: 1px solid #f3f3f3;
```

Comment:
The `background-color` property is incorrectly used here. It should be replaced with `border: 1px solid #f3f3f3;` to ensure the styling is applied as intended.